### ISSUE TITLE
feat(patch): Under videos "Load More" button removal

### ISSUE THREAD

**Issue by 0xrxL** [revanced/revanced-patches#2068]
### Application

YouTube

### Issue

After performing a search by creator name (for example: <b>Stockdroid</b>), at the bottom of some videos a button named "Load More" is shown.

### Patch

Block the non-litho component named:

<b>
- expand_button_down (type: layout)
</b>

and hide it with the method "AdRemoverAPI.HideViewWithLayout1dp(view);"

### Motivation

Once tapped, this button apparently show the most popular videos of a specific channel. These new videos are copies of videos already in the list, which could be found by scrolling the list a bit more.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by 0xrxL** [comment]
<img width="173" alt="Immagine 2023-05-03 175645" src="https://user-images.githubusercontent.com/127248639/235972782-73f2fcec-daa6-444e-8843-90c0db8b56bb.png">

**Comment by 0xrxL** [comment]
Obviously, as usual, i'm not interested to personally open a PR for this.

**Comment by johnconner122** [comment]
It's done. Suggest a name for patch. 🙂

**Comment by oSumAtrIX** [comment]
hide-load-more-button - Hides the button under videos to load similar videos

**Comment by johnconner122** [comment]
Is there a way to add a check for `search feed`, it is also hiding load more button in channel section.

![Screenshot_2023-07-03-12-22-27-469-edit_app revanced android youtube](https://github.com/revanced/revanced-patches/assets/107796137/2cbc889c-c506-4bfb-ad05-e167d381136d)
